.github/workflows/develop.yml

```diff
@@ -314,10 +314,10 @@ jobs:
           CARGO_BUILD_RUSTFLAGS: "-C link-arg=-fuse-ld=lld"
           RUST_BACKTRACE: 1
           CARGO_INCREMENTAL: 0
-          GT_S3_BUCKET: ${{ secrets.S3_BUCKET }}
-          GT_S3_ACCESS_KEY_ID: ${{ secrets.S3_ACCESS_KEY_ID }}
-          GT_S3_ACCESS_KEY: ${{ secrets.S3_ACCESS_KEY }}
-          GT_S3_REGION: ${{ secrets.S3_REGION }}
+          GT_S3_BUCKET: ${{ vars.AWS_CI_TEST_BUCKET }}
```

Comment:
This can run with PR. I suppose in that case `vars.` will be empty?

Reply:
`AWS_CI_TEST_BUCKET` and `AWS_CI_TEST_BUCKET_REGION` are the variables in GitHub runner settings. In my opinion, the bucket name and region are not secrets, and using them as internal variables is more convenient. So I change them from secrets to repo variable.

Reply:
Thanks for your explanation, SGTM.